kymatio/scattering2d/frontend/__init__.py

```diff
@@ -0,0 +1,21 @@
+__all__ = ['Scattering2D']
+
+import logging
+
+class Scattering2D(object):
+    def __init__(self, *args, **kwargs):
+        if 'frontend' not in kwargs:
+            frontend='numpy'
```

Comment:
Do we want this to be torch until version .3? Should we add a deprecation warning here?

Reply:
@eickenberg @janden

Reply:
We decided to go with torch for now, with a warning. Resolving.